Content.Server/Power/NodeGroups/PowerNet.cs

```diff
@@ -29,6 +29,7 @@ public interface IPowerNet : IBasePowerNet
     public sealed class PowerNet : BaseNetConnectorNodeGroup<IPowerNet>, IPowerNet
     {
         private PowerNetSystem? _powerNetSystem;
+        private readonly IEntityManager _entMan = IoCManager.Resolve<IEntityManager>();
```

Comment:
You shouldn't do IoC resolves in private fields because it causes threading issues.